codalab/worker/local_run/local_run_state.py

```diff
@@ -75,7 +75,6 @@ class LocalRunStage(object):
         'bundle_dir_wait_num_tries',  # Optional[int]
         'resources',  # RunResources
         'bundle_start_time',  # int
```

Comment:
What's the difference between `bundle_start_time` and `container_start_time`?

Reply:
After scanning the code, `bundle_start_time` was set at `create_run` which starts to process a new run with the given bundle and resources. However, this variable seems not used in other calculation. If this is not necessary, I'll remove it in a different ticket which can be easier to track changes.